activities/communication/communications-plan.md

```diff
@@ -25,102 +25,198 @@ We believe it’s more important to talk concretely about what we’ve already d
 
 ## Objectives
 
-Our communications should help us achieve [our current priorities](../../organization/mission.md#current-priorities).
+Our communications should help us achieve [our current priorities](../../organization/mission.md#current*priorities).
+
+We have separate goals and objectives for different audiences and fora.
 
 ## Audience
 
-Our audience is divided into:
+Our audience is individuals from potential public member organizations, codebase communities, donors, staff, and board members.
 
-* current Foundation for Public Code contributors
-* future Foundation for Public Code contributors
-* influencers
+We believe we'll find them in fora organized by/for:
 
-### Foundation contributors
+* public tech fora, for example Code4NL
+* public organization leadership, for example CommonGround or OneTeamGov
+* public organization middle management, for example iBestuur or Apolitical
+* think tanks, for example FSFE, Ford Foundation, New America Foundation,  Beeck Center for Social Impact and Innovation or Code for America
+* potential codebase communities and stakeholders - including commercial vendors
 
-We believe [these are potential contributors to the Foundation for Public Code](https://about.publiccode.net/activities/value-and-impact/user-mapping/).
+## Success criteria and indicators for target fora
 
-### Influencers
+Nota bene: none of these goals contain 'hype' or 'coolness' because being perceived as trendy or zeitgeisty /won't/ help us build the trustworthy reputation or longterm relationships we need to succed.
 
-These are domain celebrities (both individuals and organizations) in open source, government digital transformation and civic tech.
+### Public tech fora
 
-## Unique selling proposition
+Our goal for engagement with this group is:
 
-Though many specific things we do are also offered by other organizations, we're the only organization focused on open source software for public organizations that does all of this together:
+* they're aware of us and our mission, and believe this is the right thing to do
+* they engage critically with our content, use it and strengthen it
+* they talk about us with other people
 
-* is non-profit and member owned with membership only open to public organizations
-* offers codebase stewardship for different projects
-* offers productization and marketing of the codebases
-* actually provides guidance and support to help public organizations make their code open
-* isn’t tied to the EU project lifecycle
-* has a reproducible process suitable not just for cities, but all public organizations
-* is international, not national
+Success is shown these responses:
```

Comment:
This formulation is not entirely clear - do you mean "Success is shown through/by these responses" ? Or maybe more directly formulated: "These responses indicate success"